base_station/gui/src/components/IKControls.vue

```diff
@@ -50,6 +50,18 @@
                     Weight In
                 </button>
             </div>
+            <div class="input">
+                 Preset name: <input type='message' v-model ='name'>
+                 Button number: <input type='message' v-model ='buttonInc'>
```

Comment:
Can we call this buttonIdx or something related to 'index' since it's not meant for incrementing?

Reply:
Referring to the variable buttonInc